depends/packages/native_cctools.mk

```diff
@@ -52,6 +67,9 @@ endef
 
 define $(package)_stage_cmds
   $(MAKE) DESTDIR=$($(package)_staging_dir) install && \
+  mkdir -p $($(package)_staging_prefix_dir)/lib/ && \
+  cd $($(package)_extract_dir)/installed_libtapi && \
+  cp lib/libtapi.so.6 $($(package)_staging_prefix_dir)/lib/ && \
```

Comment:
We want this built statically if possible, otherwise this introduces a (rather unnecessary) LDPATH runtime requirement. Does build.sh have static lib support?